RELEASE-LEADS.md

```diff
@@ -275,7 +277,7 @@ branch of each repository respectively.
 ### Cut `release-x.y` branches of all remaining repositories
 
 After the pin PRs are merged, cut the `release-x.y` branch in each of the
-remaining repositories (except `operator`):
+remaining repositories (except `operator` and `client`):
```

Comment:
Shall we add a remark that those two are cut separately by people from the respective working group?

Reply:
Done